### PR TITLE
let users know that they will need to pay 100 BOT for event creation

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

**Comment by liviavinci** [comment]
screen shot 2018-03-24 at 11 03 50 pm

File: src/languageProvider/locales/en_US.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
   "cardInfo.topic": "Topic Created",
   "cardInfo.vote": "Voting",
   "cardInfo.withdraw": "Withdraw",
+  "create.escrowNoteTitle": "100 BOT Escrow",
```
Comment (dwalintukan) [inline src/languageProvider/locales/en_US.json]: need to use a placeholder. this number can be dynamic. change to `BOT Escrow`

File: src/languageProvider/locales/en_US.json [hunk 2]
```diff
@@ -16,6 +16,8 @@
...
+  "create.escrowNoteDesc": "You will need to pay 100 BOT as escrow in order to create an event.",
```
Comment (dwalintukan) [inline src/languageProvider/locales/en_US.json]: 100 can be a dynamic number

File: src/scenes/CreateEvent/index.js [hunk 3]
```diff
@@ -260,6 +269,13 @@ class CreateEvent extends React.Component {
         <Form onSubmit={handleSubmit(this.submitCreateEvent)}>
           <DialogTitle>{intl.formatMessage(messages.dialogTitle)}</DialogTitle>
           <DialogContent>
+            <div className={classes.importantNoteContainer}>
+              <ImportantNote
+                className={classes.createEscrowNote}
+                heading={intl.formatMessage(messages.escrowNoteTitle)}
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: use `eventEscrowAmount` prop to get the dynamic amount and use placeholder string

File: src/scenes/CreateEvent/index.js [hunk 4]
```diff
@@ -260,6 +269,13 @@ class CreateEvent extends React.Component {
...
+                message={intl.formatMessage(messages.escrowNoteDesc)}
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: use placeholder string with dynamic `eventEscrowAmount`

File: src/scenes/CreateEvent/index.js [hunk 5]
```diff
@@ -112,6 +113,14 @@ const messages = defineMessages({
     id: 'str.outcomes',
     defaultMessage: 'Outcomes',
   },
+  escrowNoteTitle: {
+    id: 'create.escrowNoteTitle',
+    defaultMessage: '100 BOT Escrow',
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: use placeholder string

File: src/scenes/CreateEvent/index.js [hunk 6]
```diff
@@ -112,6 +113,14 @@ const messages = defineMessages({
...
+  },
+  escrowNoteDesc: {
+    id: 'create.escrowNoteDesc',
+    defaultMessage: 'You will need to pay 100 BOT as escrow in order to create an event.',
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: placeholder string

**Pull request merged**